docs/howto/production/migrations.md

```
@@ -22,6 +22,13 @@ ALTER TABLE procrastinate_jobs ADD COLUMN extra TEXT;
 The migration scripts are pure-SQL scripts, meaning that they may be applied to the
 database using any PostgreSQL client, including `psql` and `PGAdmin`.
 
+Procrastinate also ships optional Alembic revisions that wrap the same SQL
+migration scripts. Install them with:
+
+```console
+$ pip install "procrastinate[alembic]"
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)